mkosi/__init__.py

```diff
@@ -1089,6 +1089,12 @@ def install_sandbox_trees(config: Config, dst: Path) -> None:
         Path(dst / "etc/crypto-policies").mkdir(exist_ok=True)
         copy_tree(p, dst / "etc/crypto-policies/back-ends", sandbox=config.sandbox)
 
+        # The files in /usr/share/crypto-policies have the .txt extension but in /etc/crypto-policies it
+        # should have the .config extension so rename all the files after copying them.
+        for p in (dst / "etc/crypto-policies/back-ends").iterdir():
+            if p.suffix == ".txt":
+                p.rename(p.with_suffix(".config"))
```

Comment:
On Fedora at least the files with correct extension live under `/usr/share/crypto-policies/back-ends/` instead of `/usr/share/crypto-policies/`